lib/src/widgets/home_route_drawer.dart

```diff
@@ -0,0 +1,232 @@
+              leading: const Icon(Icons.settings),
+              onTap: () => {
+                context.go('/in_dev'),
+                setState(() {
+                  _homeRouteTile = 5;
+                }),
+              },
+              selected: _homeRouteTile == 5,
+              title: const Text(
+                'Settings',
+                softWrap: true,
+              ),
+            ),
+            ListTile(
+              enableFeedback: true,
+              leading: const Icon(Icons.feedback),
+              onTap: () => {
+                context.go('/in_dev'),
+                setState(() {
+                  _homeRouteTile = 6;
+                }),
+              },
+              selected: _homeRouteTile == 6,
+              title: const Text(
+                'Feedback',
+                softWrap: true,
+              ),
+            ),
+            ListTile(
+              enableFeedback: true,
+              leading: const Icon(Icons.privacy_tip),
+              onTap: () => {
+                context.go('/in_dev'),
+                setState(() {
+                  _homeRouteTile = 7;
+                }),
+              },
+              selected: _homeRouteTile == 7,
+              title: const Text(
+                'Privacy Policy',
+                softWrap: true,
+              ),
+            ),
+            BrowserViewModel(
+              browserLaunchedAspect: _browserLaunched,
+              child: ListTile(
+                enableFeedback: true,
+                leading: const Icon(Icons.link),
+                onTap: () => {
+                  setState(() {
+                    _homeRouteTile = 8;
+                    _browserLaunched = _launchBrowser(launchUrl);
+                  }),
+                },
+                selected: _homeRouteTile == 8,
+                title: const Text(
+                  'Our Site',
+                  softWrap: true,
+                ),
+              ),
+            ),
+            ListTile(
+              enableFeedback: true,
+              leading: const Icon(Icons.info),
+              onTap: () => {
+                context.go('/in_dev'),
+                setState(() {
+                  _homeRouteTile = 9;
+                }),
+              },
+              selected: _homeRouteTile == 9,
+              title: const Text(
+                'About',
+                softWrap: true,
+              ),
+            ),
+            ListTile(
+              enableFeedback: true,
+              leading: const Icon(Icons.share),
+              onTap: () => {
+                context.go('/in_dev'),
+                setState(() {
+                  _homeRouteTile = 10;
+                }),
+              },
+              selected: _homeRouteTile == 10,
+              title: const Text(
+                'Share',
+                softWrap: true,
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _browserLaunchStatus(
```

Comment:
What's the purpose of this?

Reply:
Browser launcher

Reply:
But we don't need neither browser launcher, nor any connected status.

Reply:
You misunderstand. https://www.helpapaw.org is launched in the external browser when the "Our Site" menu item is clicked in the navigation drawer.

Reply:
We need just a single line for this, not a whole component. It's just launch and forget.

Reply:
It is all one component. That base widget error handles the launch status.